polymer/src/sample-category-data.html

```diff
@@ -0,0 +1,194 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../node_modules/@npm-polymer/polymer/polymer-element.html">
+<link rel="import" href="../node_modules/@npm-polymer/iron-ajax/iron-ajax.html">
+
+<dom-module id="sample-category-data">
+  <script>
+
+
+  (function() {
+
+    class SampleCategoryData extends Polymer.Element {
+
+      static get is() { return 'sample-category-data'; }
+
+      static get properties() { return {
+
+        categoryName: String,
+
+        itemName: String,
+
+        categories: {
+          type: Array,
+          notify: true
+        },
+
+        category: {
+          type: Object,
+          //computed: '_computeCategory(categoryName)',
```

Comment:
If this is unused, it can be removed as well as `_computeCategory`

Reply:
It's used in `_categoriesLoaded`